src/Feature/Editor/ContentView.re

```diff
@@ -170,6 +170,17 @@ let renderText =
           | _ => Some(List.hd(v))
           }
         };
+      let bufferLine = Editor.viewLine(editor, item).contents;
+      let startPixel =
```

Comment:
I think we might just want to use the editor `scrollX` for `startPixel` - may simplify things.

Reply:
Sounds good!

Reply:
Changed in 0b6a796!